content/04.results.md

```diff
@@ -85,3 +83,16 @@ Before calibration (green lines), our models were aligned with the ideal calibra
 Applying the temperature scaling calibration algorithm (blue lines) did not substantially improve the calibration of the model in most cases.
 The exception to this pattern is the Disease associates Gene (DaG) model where high confidence scores are shown to be better calibrated.
 Overall, calbrating deep learning models is a nontrivial task that requires  more complex approaches to accomplish.
+
+### Baseline Comparison
+
+![
+Our extractor shows comparable performance with an unsupervised baseline when predicting edges in Hetionet v1.
+We compared our model (blue) with the CoCoScore model [@tag:cocoscore](green) .
+The y axis represents AUROC and the x axis represents the edge type both models are trying to predict.
+](https://raw.githubusercontent.com/danich1/snorkeling/b6e707ea843ac5d66f62ed09277e6b1d3d4b8bf3/figures/literature_models/coco_score_auroc.png){#fig:cocoscore_comparison}
+
+Our model showed promising performance in terms of recalling edges in Hetionet v1.
+We aimed to assess our model's performance relative to an unsupervised baseline [@tag:cocoscore] by comparing both models' ability to predict preexisting edges within our test set.
```

Comment:
```suggestion
We assessed our model's performance relative to a recently published method [@tag:cocoscore].
Though our method is primarily designed to predicted assertions, not edges, we compared performance at an edge level because this was available for CoCoScore.
```